### ISSUE TITLE
Documentation: Update Form component code examples

### ISSUE THREAD

**Issue by myrta2302** [swisspost/design-system#3874]
Update form component examples in the documentation:
- [x] Add `aria-describedby` that refers to the validation message for the following components: Card Control (CSS), Checkbox, Radio Btns, Switch
- [x] Add `aria-invalid` for the invalid state of the 'Card Control' component
- [x] Align the following examples: 'Textarea', 'Input' and 'Select' examples within the Form Validation page have `aria-describedby` that refers to the validation msg, but the individual examples have `aria-describedby` that refer to the hint. Both the hint and validation msg should be referenced in all cases.

**Comment by oliverschuerch** [comment]
Depends on #3630

**Comment by myrta2302** [comment]
Updated tasks
